client/src/mock-data/mockSuggestedAssignments.js

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+const mockSuggestedAssignments = [
```

Comment:
I feel like this is quite different to the interface that the assignments will probably have but we can look at this later together: there might not be a category for assignments but it can possibly be created. Then the functions and components using the 'category' property might need to be modified somehow. For now this is fine.